client/keys/mnemonic.go

```diff
@@ -58,7 +57,7 @@ func runMnemonicCmd(cmd *cobra.Command, args []string) error {
 		// hash input entropy to get entropy seed
 		hashedEntropy := sha256.Sum256([]byte(inputEntropy))
 		entropySeed = hashedEntropy[:]
-		printStep()
+		fmt.Fprintf(os.Stderr, "-------------------------------------")
```

Comment:
Please remove all occurrences of printStep(), including the function itself:
```
$ grep -rw printStep
client/keys/add.go:func printStep() {
client/keys/mnemonic.go:		printStep()
```

I don't even why we were printing this horizontal lines in the first place...

Reply:
Please go ahead :+1: